.travis.yml

```diff
@@ -5,12 +5,13 @@ python:
   - "3.6"
 
 install:
-  - curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
-  - helm init --client-only
+  - curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get-helm-3 | bash
```

Comment:
FYI @jacobtomlinson, so that this doesn't get lost in the changes: Trying to update this to use helm3, since my charts use `apiVersion: v2`. I haven't checked to see if `apiVersion: v1` works on not.

Reply:
Ok thanks